src/java/org/apache/cassandra/audit/AuditLogEntry.java

```diff
@@ -231,6 +242,7 @@ public Builder(AuditLogEntry entry)
             operation = entry.operation;
             options = entry.options;
             state = entry.state;
+            metadata = entry.metadata;
```

Comment:
Should we use the setMetadata method that is defined below as part of this PR?

Reply:
Just making this consistent with the rest of the code block. But I think we may want to copy the map. Let me address that